imageroot/bin/DB-fix

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2025 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+# Fix row format
+if occ setupchecks | grep -q 'MySQL row format: Incorrect'; then
+  echo "Fix DB row format..."
+  # Create temp sql file
+  tmpsql=$(mktemp dbfix.XXXXXX)
+  trap 'rm -f ${tmpsql}' EXIT
+
+  # Write sql file
+  podman exec nextcloud-db mysql -u nextcloud -pnextcloud -Bse "SELECT CONCAT('ALTER TABLE \`', TABLE_NAME, '\`ROW_FORMAT=DYNAMIC;') FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'nextcloud' AND ENGINE = 'InnoDB';" > ${tmpsql}
+
+  # Execute sql file
+  podman exec -i nextcloud-db mysql -u nextcloud -pnextcloud -D nextcloud < ${tmpsql}
+fi
+
+# Fix mimetype migrations
+occ setupchecks | grep -q 'Mimetype migrations available: One' && occ maintenance:repair --include-expensive
+
+# Fix missing indices
+occ setupchecks | grep -q 'Database missing indices: Detected' && occ db:add-missing-indices
```

Comment:
hum this I thinks are common, at least I have already read them I think

Reply:
Yes, that's right but from the [issue](https://github.com/NethServer/dev/issues/7522) it's intended to also fix other warnings.